### PR TITLE
Create nomination form

### PR DESCRIPTION
**Pull request by gtm7712**
Create a form for external users to submit nominations to be considered for the National Women's Hall of Fame

### CODE REVIEW

**Comment by cg7346** [comment]
overall looks great!

File: dashboard/src/routes/(nomination-form)/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,304 @@
+<script lang="ts">
+  import {
+      Button, Checkbox, Form,
+      FormGroup, NumberInput, RadioButton, RadioButtonGroup, TextArea, TextInput
+  } from "carbon-components-svelte";
+  import "carbon-components-svelte/css/all.css";
+  import Login from "carbon-icons-svelte/lib/Login.svelte";
+  import "../../css/index.css";
+
+  // let theme = "g90";
```
Comment (cg7346) [inline dashboard/src/routes/(nomination-form)/+page.svelte:10]: here

File: dashboard/src/routes/(nomination-form)/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+  let q3ck: string[] = [];
+</script>
+
+<main>
+  <!-- <Theme bind:theme persist persistKey="__carbon-theme" />
+
+  <RadioButtonGroup legendText="Color Theme" bind:selected={theme}>
+    <RadioButton labelText="White" value="white" />
+    <RadioButton labelText="Dark" value="g90" />
+  </RadioButtonGroup> -->
```
Comment (cg7346) [inline dashboard/src/routes/(nomination-form)/+page.svelte:40]: here

**Review by cg7346 — COMMENTED** [review]
just make sure to clean up any commented-out code if you don't need it anymore

**Review by cg7346 — APPROVED** [review]

**Review by ksd4250 — APPROVED** [review]

**Pull request merged**